tests/sentry/notifications/utils/test_participants.py

```diff
@@ -1,14 +1,18 @@
+from __future__ import annotations
```

Comment:
this file was partially typing-fixed by the change so I did the rest of the fixing here